plugins/jobs/src/js/JobsRunNow.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import "rxjs/add/operator/switchMap";
+import { graphqlObservable } from "data-service";
+import gql from "graphql-tag";
+
+import defaultSchema from "./data/JobModel";
+
+const runNowJobMutation = gql`
+  mutation {
+    runJob(id: $jobId) {
+      id
+    }
+  }
+`;
+
+const runNowJobGraphql = jobId => {
+  return graphqlObservable(runNowJobMutation, defaultSchema, { jobId });
+};
+
+export default function(jobId) {
+  return {
+    label: "Run Now",
+    onItemSelect() {
+      // take(1) makes sure the observable is going to complete after finishing
+      // the request, so we don't have to care about unsubscribing.
+      runNowJobGraphql(jobId)
```

Comment:
@bstavroulakis it looks you were right here, I could not do it with just the observable, it would not trigger the mutation. Which in my rxjs book (and @orlandohohmeier ) is weird :D